src/index.js

```diff
@@ -4,15 +4,27 @@ import nodemailer from 'nodemailer'
 
 export default function (moduleOptions) {
   const options = { ...this.options.mail, ...moduleOptions }
+
   if (!options.smtp) {
     throw new Error('SMTP config is missing.')
   }
+
+  if(!options.mail || (!options.mail.to && !options.mail.cc && !options.mail.bcc)) {
+    throw new Error('You have to provide to/cc/bcc in config.')
+  }
+
   const app = express()
   const transport = nodemailer.createTransport(options.smtp)
   app.use(express.json())
   app.post('/send', async (req, res) => {
     try {
-      await transport.sendMail(req.body)
+      await transport.sendMail({
```

Comment:
use req.body but overwirte to, cc and bcc with empty (so no injection from client posible) and overwrite it with default values.

Reply:
Nice. I've simplified this a bit with optional chaining.